### DISCUSSION TITLE
Using Power voronoi in Neper

### DISCUSSION THREAD

**Discussion by kumaranvenk** [neperfepx/neper#769] [Q&A] [answered]
How to Create a microstuctures using power voronoi in Neper?? Can you suggest with the help of example. I have 150 grains and I want to use diameter as their weight to create power voronoi

**Comment by mkasemer** [comment, marked as answer]
Not sure what power Voronoi is, but certainly you can target morphological properties, such as the distribution of grain diameters: https://neper.info/generation.html#tessellations-from-morphological-properties

Otherwise, if you want to directly weight each grain based on the diameter (rather than targeting a distribution), you can do this via an external file containing weights: https://neper.info/doc/neper_t.html#cmdoption-morphooptiini
